util/translate_fight.py

```diff
@@ -5,12 +5,15 @@
 import json
 import re
 import sys
-
+import time
+import requests
 import fflogs
+import encounter_tools as e_tools
+
 
 # 'en' here is 'www' which we consider the "base" and do automatically.
 # 'cn' exists on fflogs but does not have proper translations, sorry.
```

Comment:
This comment is maybe no longer relevant?

Reply:
Perhaps? Given that we aren't actually removing the FFLogs functionality I would be hesitant to take this out as well.